config/fabric.py

```diff
@@ -66,19 +69,13 @@ def isolate(portid, namespace):
 #
 @port.command()
 @click.argument('portid', metavar='<portid>', required=True)
+@click.option('-f', '--force', is_flag=True, help='Force to unisolate a link even if it is auto isolated.')
```

Comment:
add default value as false?

Reply:
> add default value as false?

when I run the test on or command on a system , it works as it has default value of false , so I did not have it before. 
I can add it